src/data/markdown/translated-guides/en/07 Misc/09 Glossary.md

```diff
@@ -1,189 +1,203 @@
-- [Load Test](#load-test)
+- [k6 options](#k6-options)
+- [Load test](#load-test)
+- [Load zone](#load-zone)
 - [Metric](#metric)
+- [Metric sample](#metric-sample)
 - [Parameterization](#parameterization)
-- [Performance Threshold](#performance-threshold)
 - [Reliability](#reliability)
-- [Requests per Second](#requests-per-second)
+- [Requests per second](#requests-per-second)
 - [Saturation](#saturation)
 - [Scalability](#scalability)
-- [Service-level Agreement](#service-level-agreement)
-- [Service-level Indicator](#service-level-indicator)
-- [Service-level Objective](#service-level-objective)
+- [Scenario](#scenario)
+- [Service-level agreement](#service-level-agreement)
+- [Service-level indicator](#service-level-indicator)
+- [Service-level objective](#service-level-objective)
 - [Smoke test](#smoke-test)
 - [Soak test](#soak-test)
 - [Stability](#stability)
 - [Stress test](#stress-test)
 - [System under test](#system-under-test)
-- [Test configuration](#test-configuration)
-- [Test Run](#test-run)
-- [Test Script](#test-script)
-- [User Journey](#user-journey)
-- [User Scenario](#user-scenario)
-- [Vertical Scalability](#vertical-scalability)
-- [Virtual Users](#virtual-users)
+- [Test run](#test-run)
+- [Test script](#test-script)
+- [Threshold](#threshold)
+- [Throughput](#throughput)
+- [Vertical scalability](#vertical-scalability)
+- [Virtual users](#virtual-users)
 
 </Glossary>
 
-### Correlation
-
-**Correlation** is a term used for describing the process of capturing dynamic data, received from the system under test, for reuse in subsequent requests. A common use case for correlation is retrieving and reusing a session id, or token, throughout the whole lifetime of a [virtual user](#virtual-user).
-
-### Dynamic Data
+<dl>
+ 
+<dt id="application-performance-monitoring">Application performance monitoring</dt>
+<dd>*(Or APM)*. The practice of monitoring the performance, availability, and reliability of a system. When running load tests, Grafana is an example of a compatible APM service, as users can use Grafana to monitor the load test from the side of the system under test.
+<p>Read more: [k6 cloud has built-in support to export to a few APM services](https://k6.io/docs/cloud/integrations/cloud-apm/).</p></dd>
 
-**Dynamic data** is data that might, or will, change between test runs. Common examples are order ids, session tokens or timestamps.
+<dt id="concurrency">Concurrency</dt>
+<dd>When multiple computations happen at the same time.
+In the context of k6, virtual users can make concurrent requests as a test runs. In k6 cloud, you can also run multiple tests concurrently.</dd>
 
-### Endurance Testing
+<dt id="correlation">Correlation</dt>
+<dd>The process of taking [dynamic data](#dynamic-data) received from the system under test then reusing the data in a subsequent request. Testers commonly use correlation to retrieve and reuse session IDs or tokens for a [virtual user](#virtual-user)'s iteration lifetime.
```

Comment:
> Testers commonly use correlation to retrieve and reuse session IDs or tokens for a [virtual user](#virtual-user)'s iteration lifetime.

Consider removing the section above.

Consider renaming to Data correlation (Script correlation)

We have the concept of "Test result correlation" when you want to correlate testing results with the results from APM solutions.